.eslintrc.js

```diff
@@ -26,38 +26,17 @@ module.exports = {
     tick: true,
     jest: true,
   },
-  parserOptions: {
-    project: true,
-    tsConfigRootDir: __dirname,
-  },
```

Comment:
> My motivation is to tackle the typescript-eslint related issues where the file that's triggered through a pre-commit-hook isn't included in the proper tsconfig include property. Such error is:

FWIW https://typescript-eslint.io/packages/parser#experimental_useprojectservice should fix that 🙂.

Though, unless I'm missing something, I don't see any [typed rules](https://typescript-eslint.io/linting/typed-linting) enabled for files? Just rules that happen to use TypeScript syntax. So I'm wondering if you can do away `parserOptions` altogether?